core/domain/feedback_jobs_continuous.py

```diff
@@ -44,15 +46,33 @@ class FeedbackAnalyticsAggregator(jobs.BaseContinuousComputationManager):
 
     @classmethod
     def get_event_types_listened_to(cls):
+        """Get the event types that this class is subscribed to.
+
+        Returns:
+            list(str). List of event types that this class is subscribed to.
+        """
         return [feconf.EVENT_TYPE_NEW_THREAD_CREATED,
                 feconf.EVENT_TYPE_THREAD_STATUS_CHANGED]
 
     @classmethod
     def _get_realtime_datastore_class(cls):
+        """Get the realtime datastore class used by the realtime layer.
+
+        Returns:
+            FeedbackAnalyticsRealtimeModel. Datastore class used by the
+            realtime layer, which should subclass of
```

Comment:
change '... which should subclass of ...' to 'which should be a subclass of ...'